atk-sys/tests/layout.c

```diff
@@ -1,5 +1,5 @@
-// This file was generated by gir (https://github.com/gtk-rs/gir @ 5ec9961)
-// from gir-files (https://github.com/gtk-rs/gir-files @ 3fde76b)
+// This file was generated by gir (https://github.com/gtk-rs/gir @ c530bb9)
+// from gir-files (https://github.com/gtk-rs/gir-files @ a788da4)
```

Comment:
Can we switch to a single version file?

Reply:
Also it will be less effect as each sys need contain his `version.txt`